### PR TITLE
Added Support for CSV File Input

### PR DESCRIPTION
**Pull request by Hugo1104**

### CODE REVIEW

File: test/csv.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+34384,12:02:36.269827674,2,echo,(7896),>,write,fd=1(/dev/pts/3),data=hello world.,size=12,res=12
```
Comment (pkmoore) [inline test/csv.txt:1]: Small fix. Probably best to have the file called something more descriptive with a .csv extension

File: cslang/type_checker.py [hunk 2]
```diff
@@ -17,50 +17,41 @@ def check_no_output_on_not_dataword(ast):
                 print(node)
                 raise CSlangError("Output expressions are not allowed on NOT datawords")
 
-                
 
-#one of them this helper function
-#if one thing is called twice it is not top level
```
Comment (pkmoore) [inline cslang/type_checker.py:23]: You should remove your notes from PR code

File: cslang/adt.py [hunk 3]
```diff
@@ -79,10 +79,9 @@ def define_type(self, container_type, types):
             raise CSlangError(
                 "Illegal type redefinition for type: {}".format(container_type)
             )
-
```
Comment (pkmoore) [inline cslang/adt.py:82]: Probably don’t change white space in unrelated files. This can cause the build to be rejected because the auto-formatted doesn’t like it.

File: cslang/csvtodatawords.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    def handle_event(self, event):
+        if not any(
+            self.containerbuilder.top_level.values()
+        ) or not self.containerbuilder.top_level.get(event[6]):
+            return UninterestingDataWord(event)
+        else:
+            length = len(event)
+            argslist = []
+            for i in range(7,length):
```
Comment (pkmoore) [inline cslang/csvtodatawords.py:49]: Why 7+ here? Might need a comment describing what’s happening

**Comment by pkmoore** [comment]
Hey Hugo,
I’ve left some comments with a few small things to fix. Once you’ve done that use the “black” Python module to auto format the code so it’s accepted by our continuous integration setup. Let me know if you have any questions! Thanks!

**Comment by Hugo1104** [comment]
Hi Preston,

I have resolved the minor problem and reformat the code. Please let me know if there is more problem

**Comment by pkmoore** [comment]
Looks good!  Thanks!
